trees/level_order_traversal.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python
```

Comment:
This should be in `graphs`, not `trees`.

Reply:
Hey, 
I believe it's a tree traversal algorithm.